internal/version/version.go

```diff
@@ -1,32 +1,68 @@
+// Package version provides the version that the binary was built at.
 package version
 
 import (
 	"fmt"
-	"runtime"
 	"runtime/debug"
+	"strings"
+
+	enduroroot "github.com/artefactual-sdps/enduro"
```

Comment:
Did you mean to import "github.com/artefactual-sdps/enduroroot" here?  I think importing github.com/artefactual-sdps/enduro will cause a recursive import?

Reply:
This doesn't cause circular imports, I am pretty sure I would get circular import errors but I don't. Also, I modelled it the same way as sdps-api which also imports the root.